lib/fake_async.dart

```diff
@@ -12,284 +12,235 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-part of quiver.testing.async;
+import 'dart:async';
+import 'dart:collection';
 
-/// A mechanism to make time-dependent units testable.
+import 'package:collection/collection.dart';
+import 'package:quiver/time.dart';
+
+/// The type of a microtask callback.
```

Comment:
[nit] Is this comment adding information?

Reply:
I don't like leaving members undocumented. Even a trivial comment like this implies that there's no extra information that's being left out—it's essentially saying, "this is what it looks like".